stable/xray/templates/xray-analysis-deployment.yaml

```diff
@@ -88,7 +88,7 @@ spec:
           mountPath: "/scripts"
       containers:
       - name: {{ .Values.analysis.name }}
-        image: {{ .Values.analysis.image }}:{{ .Values.common.xrayVersion }}
+        image: {{ .Values.analysis.image }}:{{ default .Chart.AppVersion .Values.common.xrayVersion }}
```

Comment:
See "Configuration" section in https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md - I think this breaks this.

Reply:
@davidkarlsen There are couple of charts using this method.

Reply:
@unguiculus any viewpoints on this?

Reply:
it does not really break the `Chart Review Guidelines` as using `AppVersion` allows better to manage docker tags in multiple places